log4j-core/src/test/java/org/apache/logging/log4j/core/time/internal/format/FastDateParserTest.java

```diff
@@ -628,12 +627,12 @@ private static Calendar initializeCalendar(final TimeZone tz) {
         return cal;
     }
 
-    private static enum Expected1806 {
+    private enum Expected1806 {
         India(INDIA, "+05", "+0530", "+05:30", true), 
         Greenwich(GMT, "Z", "Z", "Z", false), 
         NewYork(NEW_YORK, "-05", "-0500", "-05:00", false);
 
-        private Expected1806(final TimeZone zone, final String one, final String two, final String three, final boolean hasHalfHourOffset) {
+        Expected1806(final TimeZone zone, final String one, final String two, final String three, final boolean hasHalfHourOffset) {
```

Comment:
Why not `private`?

Reply:
Same case above. In my opinion it's redundant.